[ID] Update `Beatmap category` (#14793)
ppy/osu-wiki: wiki/Beatmap/Category/id.md

## Kategori yang ada saat ini

### Graveyard {id=graveyard}

Graveyard adalah tempat peristirahatan bagi beatmap yang sudah diabaikan oleh [pembuatnya](/wiki/Beatmap/Beatmap_host). Beatmap Graveyard tidak memiliki papan peringkat, namun akan tetap bisa diunduh, dimainkan, dan berkontribusi ke [statistik permainan](#statistik-beatmap-non-ranked) yang ada. Beatmap dalam kategori ini ditandai dengan simbol tanda tanya (![](/wiki/shared/status/graveyard.png)) pada [layar pemilihan lagu](/wiki/Client/Interface#song-select).

Suatu beatmap akan secara otomatis dipindahkan dari [Work in Progress dan Pending](#wip-and-pending) ke Graveyard setelah tidak diperbarui selama 4 minggu. Pemilik beatmap ini bisa "memulihkan" beatmap ini (memindahkannya kembali ke [Pending](#wip-and-pending)) dengan cara memperbarui beatmap melalui [Beatmap Submission System](/wiki/Beatmapping/Beatmap_submission). Beatmap yang ada di Graveyard tidak akan dihitung terhadap [batas jumlah unggahan pengguna](/wiki/osu!supporter#increased-limits), namun agar beatmap ini bisa dipulihkan, pemilik beatmap ini harus memiliki ruang Pending yang kosong.

### Work in Progress dan Pending {id=wip-and-pending}

> Work in Progress (*WIP*) dan Pending adalah dua kategori yang diberikan kepada beatmap pada saat pertama kali diunggah. Beatmap dalam kedua kategori ini bisa berpindah ke kategori [Qualified](#qualified) apabila beatmap ini [dinominasikan](/wiki/Beatmap_ranking_procedure#nominasi) oleh setidaknya dua orang anggota [Beatmap Nominator](/wiki/People/Beatmap_Nominators). Proses ini pada umumnya mencakup pengkajian yang mendalam oleh para [modder](/wiki/Modding/Modder) melalui tahapan [modding](/wiki/Modding). Beatmap Work in Progress dan Pending tidak memiliki papan peringkat, namun akan tetap berkontribusi ke [statistik permainan](#statistik-beatmap-non-ranked) yang ada. Beatmap dalam kedua kategori ini ditandai dengan simbol tanda tanya (![](/wiki/shared/status/graveyard.png)) pada layar pemilihan lagu.

Reviewer: ```suggestion
Work in Progress (*WIP*) dan Pending adalah dua kategori yang diberikan kepada beatmap pada saat pertama kali diunggah. Beatmap dalam kedua kategori ini bisa berpindah ke kategori [Qualified](#qualified) apabila beatmap ini [dinominasikan](/wiki/Beatmap_ranking_procedure#nominasi) oleh setidaknya dua orang anggota [Beatmap Nominator](/wiki/People/Beatmap_Nominators). Proses ini biasanya mencakup banyak masukkan dari para [modder](/wiki/Modding/Modder) melalui tahapan [modding](/wiki/Modding). Beatmap Work in Progress dan Pending tidak memiliki papan peringkat, namun akan tetap berkontribusi ke [statistik permainan](#statistik-beatmap-non-ranked) yang ada. Beatmap dalam kedua kategori ini ditandai dengan simbol tanda tanya (![](/wiki/shared/status/graveyard.png)) pada layar pemilihan lagu.
```

Beatmap Qualified memiliki papan peringkat, namun tidak memberikan [performance point](/wiki/Performance_points) dan semua skor yang ada akan dihapus pada saat beatmap ini keluar dari Qualified. Beatmap dalam kategori ini ditandai dengan simbol centang (![](/wiki/shared/status/qualified.png)) pada layar pemilihan lagu.

### Ranked {id=ranked}

Beatmap Ranked adalah beatmap yang dijunjung sebagai konten resmi, dalam artian bahwa beatmap ini sudah memenuhi atau melampaui standar [kriteria ranking](/wiki/Ranking_criteria). Beatmap dalam kategori ini memungkinkan para pemain untuk saling bersaing pada papan peringkat dan mendapatkan [performance point](/wiki/Performance_points) dari skor yang dicapai. Sekalinya suatu beatmap mencapai status Ranked, status beatmap ini tidak akan lagi bisa berubah kecuali pada kasus-kasus tertentu.

Beatmap Ranked ditandai dengan simbol pangkat ganda (![](/wiki/shared/status/ranked.png)) pada layar pemilihan lagu.

### Loved {id=loved}

*Untuk informasi lebih lanjut seputar sejarah kategori Loved, kunjungi: [Sejarah Loved](/wiki/History_of_osu!/History_of_Loved)*

> Beatmap Loved adalah beatmap yang bisa jadi tidak memenuhi [kriteria ranking](/wiki/Ranking_criteria), namun *dicintai (loved)* oleh komunitas secara luas, dan oleh karenanya diangkat ke kategori ini secara berkala berdasarkan hasil pemungutan suara [Project Loved](/wiki/Community/Project_Loved). Beatmap Loved memiliki papan peringkat, namun tidak memberikan [performance point](/wiki/Performance_points) dan seluruh skor yang ada akan dihapus apabila beatmap ini dilepas dari kategori Loved. Beatmap dalam kategori ini ditandai dengan simbol hati (![](/wiki/shared/status/loved.png)) pada layar pemilihan lagu.

Reviewer: ```suggestion
Beatmap Loved adalah beatmap yang bisa jadi tidak memenuhi [kriteria ranking](/wiki/Ranking_criteria), namun *dicintai (loved)* oleh komunitas secara luas, dan oleh karenanya diangkat ke kategori ini secara berkala berdasarkan hasil pemungutan suara di [Project Loved](/wiki/Community/Project_Loved). Beatmap Loved memiliki papan peringkat, namun tidak memberikan [performance point](/wiki/Performance_points) dan seluruh skor yang ada akan dihapus apabila beatmap ini dilepas dari kategori Loved. Beatmap dalam kategori ini ditandai dengan simbol hati (![](/wiki/shared/status/loved.png)) pada layar pemilihan lagu.
```

## Sejarah

### Approved {id=approved}

Approved adalah kategori yang digunakan antara tahun 2008 dan 2014 untuk mewadahi beatmap yang durasi dan skor maksimumnya melebihi dari apa yang diizinkan oleh [kriteria ranking](/wiki/Ranking_criteria) pada saat itu, beserta dengan beatmap-beatmap tertentu yang melanggar kriteria lainnya.

Beatmap Approved pada awalnya tidak berpengaruh ke peringkat skor (*score ranking*) pemain hingga osu! berpindah dari sistem peringkat skor ke sistem [performance point](/wiki/Performance_points). Setelah adanya perubahan ini, kategori Approved kemudian dikhususkan bagi beatmap yang memiliki [waktu permainan bersih](/wiki/Beatmap/Drain_time) (*drain time*) di atas 6 menit.

Beatmap dalam kategori ini ditandai dengan simbol centang (![](/wiki/shared/status/approved.png)) pada layar pemilihan lagu dan berfungsi sebagaimana beatmap Ranked pada umumnya.

### Kemunculan kategori Qualified

> Kategori [Qualified](#qualified) diperkenalkan pada bulan Agustus 2014 pada saat [restrukturisasi BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) untuk membuat agar beatmap yang baru Ranked menjadi lebih terlihat. Apabila suatu beatmap Qualified perlu didiskualifikasi dan dikembalikan ke Pending, statistik pemain tidak akan terpengaruh, karena sejak awal kategori ini sudah dirancang untuk tidak mencatat skor secara permanen. Kemunculan kategori ini berhasil meluruskan pandangan yang ada seputar proses ranking serta mengurangi tanggapan negatif dari komunitas seputar penganuliran status Ranked (*unrank*) pada beatmap.

Reviewer: ```suggestion
Kategori [Qualified](#qualified) diperkenalkan pada bulan Agustus 2014 pada saat [restrukturisasi BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) untuk membuat agar beatmap yang baru Ranked menjadi lebih terlihat. Apabila suatu beatmap Qualified perlu didiskualifikasi dan dikembalikan ke Pending, statistik pemain tidak akan terpengaruh, karena kategori ini sendiri memang sejak awal dirancang untuk memberikan skor sementara. Kemunculan kategori ini berhasil meluruskan pandangan yang ada seputar proses ranking serta mengurangi tanggapan negatif dari komunitas seputar penganuliran status Ranked (*unrank*) pada beatmap.
```
